### ISSUE TITLE
Add keyboard shortcuts (beside Enter & Esc)

### ISSUE THREAD

**Issue by scekics** [mkheck/DialogFX#3]
I think it would be nice to  have an option to (automatically) assign keyboard shortcuts to the Buttons depending on their Label. For example 'Y' for Yes, 'N' for no etc... Please keep in mind that Labels can be localized, so these shortcuts cannot be fixed.

**Comment by mkheck** [comment]
Good idea - it's now on the list! Let me see what I can do with this. Thanks much!

**Comment by MahmoudMabrok** [comment]
I want to solve this issue , 
but i do not know what is the mean of ("Labels can be localized")
